hack/dci-merge.sh

```diff
@@ -54,100 +45,43 @@ send_status() {
-            fi
-        fi
-
-        if [[ libvirt =~ $SUPPORTED_HINTS ]] && grep -qi "^\s*Test-Hints:\s*libvirt\s*" <<< "$DESC"; then
-            VIRT=--virt
-        fi
-
-        if [[ no-check =~ $SUPPORTED_HINTS ]] && grep -qi "^\s*Test-Hints:\s*no-check\s*" <<< "$DESC"; then
+        if [[ no-check =~ $SUPPORTED_HINTS ]] && grep -qEi "^\s*Test-Hints?:\s*no-check\s*" <<< "$DESC"; then
             NB_NOCHECK=$((NB_NOCHECK+1))
             continue
         fi
 
-        if [[ force-check =~ $SUPPORTED_HINTS ]] && grep -qi "^\s*Test-Hints:\s*force-check\s*" <<< "$DESC"; then
-            FORCE_CHECK=--force-check
+        if [[ force-check =~ $SUPPORTED_HINTS ]] && grep -qEi "^\s*Test-Hints?:\s*force-check\s*" <<< "$DESC"; then
+            FORCE_CHECK=1
         fi
 
-        if [[ upgrade =~ $SUPPORTED_HINTS ]] && grep -qi "^\s*Test-Upgrade-Hints:\s*yes\s*" <<< "$DESC"; then
-            UPGRADE=--upgrade
-
-            if [ -z "$VIRT" ]; then
-                VIRT=--virt
-            fi
-
-            # process Test-Upgrade-Args-Hints
-            if [[ upgrade-args =~ $SUPPORTED_HINTS ]] && grep -qi "^\s*Test-Upgrade-Args-Hints:" <<< "$DESC"; then
-                UPGRADE_ARGS="$(sed -n -e 's/^\s*Test-Upgrade-Args-Hints:\s*//pi' <<< "$DESC")"
-            fi
-
-            # process Test-Upgrade-From-Topic-Hints
-            if [[ upgrade-from-topic =~ $SUPPORTED_HINTS ]] && grep -qi "^\s*Test-Upgrade-From-Topic-Hints:" <<< "$DESC"; then
-                UPGRADE="$UPGRADE --from-topic $(sed -n -e 's/^\s*Test-Upgrade-From-Topic-Hints:\s*//pi' <<< "$DESC")"
-            fi
-
-            # process Test-Upgrade-To-Topic-Hints
-            if [[ upgrade-to-topic =~ $SUPPORTED_HINTS ]] && grep -qi "^\s*Test-Upgrade-To-Topic-Hints:" <<< "$DESC"; then
-                UPGRADE="$UPGRADE --to-topic $(sed -n -e 's/^\s*Test-Upgrade-To-Topic-Hints:\s*//pi' <<< "$DESC")"
-            fi
+        # extract TestBos2 commands
+        if grep -qE "^\s*TestBos2:\s*" <<< "$DESC"; then
+            # shellcheck disable=SC2001,SC2086
+            CMD+=("$(sed -e 's/^\s*TestBos2:\s*//' <<< $DESC)")
         fi
-
-        if [ -n "$VIRT" ]; then
-            # process Test-Args-Hints
-            if [[ args =~ $SUPPORTED_HINTS ]]; then
-                OPTS=$(sed -n -e "s/^\s*Test-Args-Hints:\s*//pi" <<< "$DESC")
-            else
-                OPTS=
-            fi
-
-            # process Test-App-Hints
-            if [[ app =~ $SUPPORTED_HINTS ]]; then
-                APP_NAME=$(sed -n -e "s/^\s*Test-App-Hints:\s*//pi" <<< "$DESC")
-            fi
-
-            # process Test-App-Args-Hints
-            if [[ app-args =~ $SUPPORTED_HINTS ]]; then
-                APP_ARGS=$(sed -n -e "s/^\s*Test-App-Args-Hints:\s*//pi" <<< "$DESC")
-            fi
-
-            # stop at the first valid Test-Hints: string
-            break
+        # extract TestBos2Sno commands
+        if grep -qE "^\s*TestBos2Sno:\s*" <<< "$DESC"; then
```

Comment:
I'm wondering where are these TestBos2Sno/TestBos2Baremetal vars defined in the docs, just to have clear how to use them. Are they similar to TestWorkload or TestDallas?